admin/tools/docker/deployment/parallel/run.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/sh
+
+# Launch Docker containers for Qserv master and workers
+
+# @author Fabrice Jammes IN2P3
+
+set -e
+
+DIR=$(cd "$(dirname "$0")"; pwd -P)
+
+. "${DIR}/env.sh"
+
+if [ -n "$HOST_LOG_DIR" ]; then
+    LOG_VOLUME_OPT="--volume $HOST_LOG_DIR:/qserv/run/var/log"
+fi
+if [ -n "$HOST_DATA_DIR" ]; then
+    DATA_VOLUME_OPT="--volume $HOST_DATA_DIR:/qserv/data"
+fi
+
+echo
+echo "Check for running Qserv containers"
+echo "=================================="
+echo
+IS_UP="DOCKER_IS_UP"
+shmux -Bsm -S all -c "docker inspect --format='{{ .State.Running }}' ${CONTAINER_NAME} && echo ${IS_UP}" "$MASTER" $WORKERS \
+    2> /dev/null \
+    | grep "${IS_UP}" \
+    && echo "ERROR: Qserv containers are already running on nodes listed above"  && exit 1
```

Comment:
We got some false positives from this code. docker ps would show nothing running, but run.sh would exit saying DOCKER_IS_UP on all the hosts.  Running killall.sh seems to cleanup the issue.